stubs/python/build.gradle.kts

```diff
@@ -86,7 +85,7 @@ tasks {
 
     val generateProto = named<org.curioswitch.gradle.protobuf.tasks.GenerateProtoTask>("generateProto")
     generateProto.configure {
-        dependsOn(fillRunProtocScript, prepareSetupPy, ":toolsSetupMiniconda2Build")
```

Comment:
You probably need to depend on SetupMinicondaBuild now. Without any dependency the build won't work with a clear cache I think

Reply:
Ah thanks, was looking for the other instance I removed!